src/transformers/tape.js

```diff
@@ -78,8 +78,6 @@ const tPropertiesUnsupported = new Set([
     'notDeepLooseEqual',
     'notLooseEqual',
     'notLooseEquals',
-
-    'skip',
```

Comment:
Because it is also caught by https://github.com/aaronmcadam/jest-codemods/blob/c9e7dfac585d6f06b168890383ab9326213b3ec6/src/transformers/tape.js#L147

We are just lucky that the error message is the same. So you can add the `skip` back to `tPropertiesUnsupported`, and then we are good to go. :) 

Thanks for investigating